zerver/worker/queue_processors.py

```diff
@@ -391,3 +393,24 @@ def consume(self, event):
         logging.info("TestWorker should append this message to %s: %s" % (fn, message))
         with open(fn, 'a') as f:
             f.write(message + '\n')
+
+@assign_queue('embed_links')
+class FetchLinksEmbedData(QueueProcessingWorker):
+    def consume(self, event):
+        # type: (Mapping[str, Any]) -> None
+        for url in event['urls']:
+            url_preview.get_link_embed_data(url)
+
+        message = Message.objects.get(id=event['message_id'])
+        if message.content != event['message_content']:
```

Comment:
Add a comment here explaining that this check is here to handle message edits.

Ideally, we'd also have a test verifying the message edit logic.